staging/traefik-forward-auth/templates/secret.yaml

```diff
@@ -6,6 +6,8 @@ type: Opaque
 data:
   clientSecret: {{ required "traefikForwardAuth.clientSecret is required" .Values.traefikForwardAuth.clientSecret.value | b64enc | quote }}
   secret: {{ required "traefikForwardAuth.secret is required" .Values.traefikForwardAuth.secret | b64enc | quote }}
+
+{{- if .Values.traefikForwardAuth.opsPortalSecret.enable }}
```

Comment:
@jongiddy @mhrabovcin this sounds like an appropriate change to me. Does it also seem to you ?

Reply:
Putting knowledge of the Ops Portal into a chart for a different service seems strange to me. I expect a chart to be generic. Given that this has been done anyway, I don't object to this particular change.

Reply:
okay, I am thinking about it ☕️...  Let's delete this secret here and so the configuration. We don't need it here.